File: Python/136.只出现一次的数字.py
class Solution(object):
    def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums) == 0:
            return -1
        res = 0
        for num in nums:
            res = res^num
        return res

File: CPP/136.只出现一次的数字.cpp
class Solution {
public:
//通过异或运算，得出唯一出现一次的结果
    int singleNumber(vector<int>& nums) {
        if(nums.size() == 0) return -1;
        int res = 0;
        for(auto num :nums){
            res^=num;
        }
        return res;
    }
};